src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -1151,6 +1151,7 @@ private BlobHandle GetMethodSignatureHandleAndBlob(IMethodReference methodRefere
                 return existing.Key;
             }
 
+            Debug.Assert(methodReference.CallingConvention != CallingConvention.Invalid);
```

Comment:
This seems fine, but there don't appear to be any emit tests added, so I'm wondering how you ended up adding this?

Reply:
> This seems fine, but there don't appear to be any emit tests added, so I'm wondering how you ended up adding this?

I added a new option to the CallingConvention, and I wanted to verify we never attempted to emit something that was invalid. There's nothing that prompted me to add this other than that all current elements in CallingConvention are valid conventions.

---
In reply to: [359009433](https://github.com/dotnet/roslyn/pull/40431#discussion_r359009433) [](ancestors = 359009433)